legislators-social-media.yaml

```diff
@@ -4189,6 +4195,11 @@
     youtube: RepSanfordSC
     youtube_id: UCS33edNAxXe2pyFbnUUcZWA
     twitter_id: 1499993378
+- id:
+    bioguide: S001203
+    govtrack: 412742
+  social:
+    twitter: SenTinaSmith
```

Comment:
Sen. Smith doesn't have an official site yet, but the Verified checkmark, the bio, and the newness of the account are enough for me to accept this as the Senator's official account. :+1: